centreon/tests/e2e/features/Cloud-notifications/02-notification-duplicate.feature

```diff
@@ -0,0 +1,28 @@
+@ignore
+@REQ_MON-24180
+Feature: Duplicating a notification rule
+  As a Centreon user with access to the notification rules page
+  I need to duplicate a notification rule
+  So that I can save time and configure a new one without starting from scratch
+
+  Background:
+    Given a user with access to the notification rules page
+    And the user is on the notification rules page
+    And a notification rule is already created
+
+  Scenario: Duplicating a notification rule
+    When the user selects the duplication action on a notification rule
+    And the user enters a new name
+    And the user confirms
+    Then a success message is displayed
+    And then duplicated notification rule is displayed in the listing
```

Comment:
Is displaying the rule in the listing the real objective of duplication, or is it to have a real rule duplicate. What if the rule duplicate is fully empty?

Reply:
The duplication copy / paste the same rule